src/components/myExercises/index.js

```diff
@@ -0,0 +1,26 @@
+import Card from "../card";
+import Button from "../button";
+import "./style.css"
+
+const MyExercises = () => {
+    return (
+        <>
+            <Card>
+                <h3 className="my-cohort-exercises--header">My Exercises</h3>
+                <div>
+                    <div className="my-cohort-exercises--stats">
+                        <p>Modules:</p> <span>2/7 completed</span>
+                        <p>Units:</p> <span>4/10 completed</span>
+                        <p>Exercises:</p> <span>34/58 completed</span>
+                    </div>
+                
+                    <div className="my-cohort-excercises--button">
```

Comment:
(*definitely not my fault*) This should be changed to "...-exercises--stats"

Reply:
This has been handled thanks